test/decorator.test.js

```diff
@@ -939,3 +939,32 @@ test('plugin required decorators', async t => {
   app.register(plugin2)
   await app.ready()
 })
+
+test('decorateRequest/decorateReply empty string', t => {
+  t.plan(7)
+  const fastify = Fastify()
+
+  fastify.decorateRequest('test', '')
```

Comment:
Can you add a test where the initial decoration is set to `undefined`?

Reply:
I have added the test.